renderer/utils/environment.ts

```diff
@@ -0,0 +1,13 @@
+import electron from 'electron';
+
+export const isDevelopment = () => {
```

Comment:
Since this is duplicating code, why not use ipc to send this information from the main to renderer?